skills/cli/SKILL.md

```
@@ -98,10 +94,64 @@ npx bt generate
+npx bt generate --yes
+```
+
+### 7. Purge unused keys
+
+```sh
+npx bt purge
+```
+
+This finds translation keys that no longer appear in any `t("...")` call and removes them from all locale files. It asks you to confirm each key before removing it:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)